web/components/answers/answers-graph.tsx

```diff
@@ -108,6 +108,7 @@ export const AnswersGraph = memo(function AnswersGraph(props: {
         }}
         colors={{ scheme: 'pastel1' }}
         pointSize={0}
+        curve='stepAfter'
```

Comment:
Sorry, could you change to double quotes, please! (Or, run prettier, our auto-formatter.)

Reply:
Sure, done!